core/src/main/resources/jenkins/model/Jenkins/systemInfo.jelly

```diff
@@ -31,123 +31,86 @@ THE SOFTWARE.
                 <tr>
-                  <td>
-                    <st:out value="${p.shortName}"/>
-                  </td>
-                  <td>
-                    <st:out value="${p.version}"/>
-                  </td>
-                  <td>
-                    <st:out value="${state}"/>
+                  <td colspan="3">
+                    ${%No plugins installed.}
                   </td>
                 </tr>
-              </j:forEach>
-            </j:otherwise>
-          </j:choose>
-
-        </tbody>
-      </table>
-      <div class="jenkins-app-bar">
-        <div class="jenkins-app-bar__content">
-          <h1>${%Memory Usage}</h1>
-        </div>
+              </j:when>
+              <j:otherwise>
+                <j:forEach var="p" items="${app.pluginManager.plugins}">
+                  <tr>
+                    <td>
+                      <st:out value="${p.shortName}"/>
+                    </td>
+                    <td>
+                      <st:out value="${p.version}"/>
+                    </td>
+                    <td>
+                      ${p.enabled ? "%Yes" : "%No"}
+                    </td>
+                  </tr>
+                </j:forEach>
+              </j:otherwise>
+            </j:choose>
+          </tbody>
+        </table>
       </div>
-      <j:set var="type" value="${request.getParameter('type')}" />
-      <j:choose>
-        <j:when test="${type == 'sec10'}" />
-        <j:when test="${type == 'min'}" />
-        <j:when test="${type == 'hour'}" />
-        <j:otherwise>
-          <j:set var="type" value="min" />
-        </j:otherwise>
-      </j:choose>
 
-      <div>
-        ${%Timespan}:
-        <j:choose>
-          <j:when test="${type != 'sec10'}">
-            <a href="?type=sec10">${%Short}</a>
-          </j:when>
-          <j:otherwise>
-            ${%Short}
-          </j:otherwise>
-        </j:choose>
-        <st:nbsp />
-        <j:choose>
-          <j:when test="${type != 'min'}">
-            <a href="?type=min">${%Medium}</a>
-          </j:when>
-          <j:otherwise>
-            ${%Medium}
-          </j:otherwise>
-        </j:choose>
-        <st:nbsp />
-        <j:choose>
-          <j:when test="${type != 'hour'}">
-            <a href="?type=hour">${%Long}</a>
-          </j:when>
-          <j:otherwise>
-            ${%Long}
-          </j:otherwise>
-        </j:choose>
+      <div class="jenkins-tab-pane">
+        <h2 class="jenkins-tab-pane__title">${%Memory Usage}</h2>
+
+        <f:entry title="${%Timespan}" class="jenkins-form-item--small">
+          <div class="jenkins-select">
+            <select id="timespan-select" class="jenkins-select__input">
+              <option value="sec10">${%Every ten seconds}</option>
+              <option value="min" selected="true">${%Every minute}</option>
+              <option value="hour">${%Every hour}</option>
```

Comment:
I always understood timespan to refer to the total time displayed in the graph (6 hrs, 20 hrs, and a month or so?), rather than the update interval/granularity. Was this wrong? If not, do the new labels still make sense?

Reply:
Hmm I'm not too sure actually. I've restored the original labels for the time being, would anybody from @jenkinsci/sig-ux know the answer to this?